core-bundle/src/Command/MigrateCommand.php

```diff
@@ -85,6 +86,22 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $this->io = new SymfonyStyle($input, $output);
 
+        if ('json' === $input->getOption('format')) {
+            $data = [
+                'runonceFiles' => $this->getRunOnceFiles(),
+                'migrations' => $this->migrations->getPendingNames(),
+                'schema' => $this->installer->getCommands(false),
+            ];
+
+            $this->io->write(json_encode($data));
+
+            return 0;
+        }
+
+        if ('text' !== $input->getOption('format')) {
+            throw new \LogicException('Invalid format: '.$input->getOption('format'));
```

Comment:
```suggestion
            $this->io->error('Invalid format: '.$input->getOption('format'));
            
            return 1;
```

Reply:
Not sure what's right, I've copied this from here: https://github.com/contao/contao/blob/4.x/core-bundle/src/Command/UserListCommand.php#L86